FANewChanges.json

```diff
@@ -0,0 +1,304 @@
+{
+  "All": "Todo",
+  "App": "Aplicación",
+  "A store represents a company or a unique catalog of products. If your OMS is connected to multiple eCommerce stores selling different collections of products, you may have multiple Product Stores set up in HotWax Commerce.": "Una tienda representa una empresa o un catálogo único de productos. Si tu OMS está conectado a múltiples tiendas de comercio electrónico que venden diferentes colecciones de productos, es posible que tengas varias Tiendas de Productos configuradas en HotWax Commerce.": "{facilityName} no tiene ningún pedido completado en este momento.",
+  "doesn't have any orders in progress right now.": "{facilityName} no tiene ningún pedido en curso en este momento.",
+  "doesn't have any outstanding orders right now.": "{facilityName} no tiene ningún pedido pendiente en este momento.",
+  "Add Box": "Agregar Caja",
+  "Add custom field": "Agregar campo personalizado",
+ "Additional documents": "Documentos adicionales",
+  "Address 1": "Dirección 1",
+  "Address 2": "Dirección 2",
+  "Apply": "Aplicar",
+  "Authenticating": "Autenticando",
+  "Are you sure you want to change the time zone to?": "¿Estás seguro/a de que quieres cambiar la zona horaria a?",
+  "Are you sure you want to delete this CSV mapping? This action cannot be undone.": "¿Estás seguro/a de que quieres eliminar este mapeo CSV? Esta acción no se puede deshacer.",
+  "Are you sure you want to update this CSV mapping? This action cannot be undone.": "¿Estás seguro/a de que quieres actualizar este mapeo CSV? Esta acción no se puede deshacer.",
+  "Are you sure you want to perform this action?": "¿Estás seguro/a de que quieres realizar esta acción?",
+  "Are you sure you want to reject outstanding order(s)?": "¿Estás seguro/a de que quieres rechazar { ordersCount } pedido(s) pendiente(s)?",
+  "Are you sure you want to reject in progress order(s)?": "¿Estás seguro/a de que quieres rechazar { ordersCount } pedido(s) en curso?",
+  "Assign Pickers": "Asignar Recolectores",
+  "Box added successfully": "Caja agregada exitosamente",
+  "Boxes": "Cajas",
+  "Cancel": "Cancelar",
+  "Click the backdrop to dismiss.": "Haz clic en el fondo para cerrar.",
+  "Change": "Cancelar",
+  "Changes to the CSV mapping has been saved.": "Los cambios en el mapeo CSV se han guardado.",
+  "Choose language": "Elegir idioma",
+  "City": "Ciudad",
+  "Completed": "Completado",
+  "Configuration Missing": "Configuración Faltante",
+  "Confirm": "Confirmar",
+  "Configure the order fulfillment capacity of your facility.": "Configura la capacidad de cumplimiento de pedidos de tu instalación.",
+  "Control whether the store's inventory should be made available for online sales or not.": "Controla si el inventario de la tienda debe estar disponible para ventas en línea o no.",
+  "Copied": "Copiado {texto}",
```

Comment:
Revert this variable name change.

Reply:
which exact variable are we talking about here, I mostly see each of them has translations.

Reply:
`texto` should be `text`, anything that's inside `{}` are variables and not need to be changed.